### DISCUSSION TITLE
Error generating keypair: TextEncoder is not defined

### DISCUSSION THREAD

**Discussion by robertst-bitgo** [openpgpjs/openpgpjs#1254] [Q&A]
Using the 5.0.0 version, I get an exception when trying to generate keys in Node.js:
``` Error generating keypair: TextEncoder is not defined``` 

This is most likely the result of commit ab419179bcc37b1dd598b3cf041b7b200d6f9208 (https://github.com/openpgpjs/openpgpjs/commit/ab419179bcc37b1dd598b3cf041b7b200d6f9208)

**Comment by twiss** [comment]
Hi :wave: Yeah. Node.js 10 is about 2 months from being End-of-Life: https://nodejs.org/en/about/releases/. Please switch to a newer version. If you can't, you can keep using OpenPGP.js v4 for now, but Node.js 10 (and OpenPGP.js v4) will be unsupported soon.

**Reply by markwilx** [reply]
When calling `await privateKey.decrypt(passphrase)` I'm getting the error `ReferenceError: TextEncoder is not defined`

My code broke when I updated from OpenPGP v4.10.1 to v5.0.0-1.  I rewrote my function to decrypt string data and followed the example at [https://www.npmjs.com/package/openpgp#encrypt-and-decrypt-string-data-with-pgp-keys](https://www.npmjs.com/package/openpgp#encrypt-and-decrypt-string-data-with-pgp-keys).

After finding this discussion, I updated my node.js version.  I had been running node v10.19.0.  I updated to node v14.16.1 and npm v7.10.0.  I ran `npm update`, deleted `node-modules` and deleted the lock file.  I then ran `npm install`.  Afterward, I continue to receive this error.

Do you have any insights as to why I'm still getting the TextEncoder not define error?
